tests/modules/specs/_createrepo_c_modularity_hack.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
```

Comment:
Please can you explain what it does and how to use it?

Reply:
@dmach ^^

Reply:
Let's add following comment to the file:
"""
Createrepo_c doesn't support indexing module metadata.
This script indexes all yaml files found in a target directory, concatenates them and injects into repodata as "modules" mdtype.
"""

Reply:
Fixed.